themes/notmyidea/static/css/main.css

```diff
@@ -186,16 +186,18 @@ img.left, figure.left {float: left; margin: 0 2em 2em 0;}
 		font-size: 1.143em;
 		height: 40px;
 		line-height: 30px;
-		margin: 0 auto 2em auto;
-		padding: 0;
 		text-align: center;
 		width: 900px;
+		}
 		
-		border-radius: 5px;
```

Comment:
Did you test for cross-browser compatibility? I haven't had a chance to test this yet but there's some engine-specific stuff in here you've removed.